packages/dara-core/js/shared/interactivity/use-variable.tsx

```diff
@@ -81,16 +81,18 @@ export function useVariable<T>(variable: Variable<T> | T): [value: T, update: Di
     }
 
     const recoilState = getOrRegisterPlainVariable(variable, WsClient, taskContext, extras);
+    if (!isDerivedVariable(variable.default)) {
```

Comment:
We probably still need the effect  (or a similar one) below to run in this branch as well, since our internal products rely on the bus event below being fired for all variable changes. E.g. you could add

```tsx
const [value, setValue] = useRecoilState(recoilState);
useEffect(() => {
 bus.publish(...value...)
}, [value]);
return [value, setValue];
```